Packs/Twitter/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### New: Twitter v2
+Twitter integration provides access to searching recent Tweets (in last 7 days) and user information using Twitter v2 API (Available from Cortex XSOAR 6.5.0).
+##### Twitter (Deprecated)
+Deprecated. Use Twitter v2 instead.
```

Comment:
I think you should add a section for the commands that have been deleted or replaced.  Something like this:
### Commands 
#### The following commands were removed in Twitter v2:
* ***twitter-get-user-info*** - this command was removed.
* ***twitter-get-users*** - this command was replaced by ***twitter-user-get***.
* ***twitter-get-tweets*** - this command was replaced by ***twitter-tweet-search***.